website/versioned_docs/version-5.x/migrating-existing-spas.md

```diff
@@ -7,7 +7,7 @@ sidebar_label: Migrating existing code
 If you're interested in migrating existing SPAs into a single-spa, you'll
 need to do three things:
 
-1. Create a [single spa config](configuration.md)
+1. Create a [single spa config](./configuration)
```

Comment:
I'm not sure why my original fix didn't work. I tried a few different things and I couldn't get it to work with the `.md` at the end, even though my understanding was that was necessary for relative references.

In the end this works and my .md fix was wrong.

Reply:
🤷‍♂ idk either